.changes/next-release/api-change-backupstorage-5831.json

```
@@ -0,0 +1,5 @@
+{
+  "type": "api-change",
+  "category": "backupstorage",
```

nit - formatting fix
```suggestion
  "category": "``backupstorage``",
```